TriblerGUI/qt_resources/mainwindow.ui

```diff
@@ -8794,7 +8794,7 @@ border-top: 1px solid #555;
                                             margin: 10px;</string>
                </property>
                <property name="text">
-                <string>Trust statistics</string>
+                <string>Trust Display</string>
                </property>
```

Comment:
As discussed we conform to the upper case convention. So 'Trust Display' in this case.